content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -22,6 +22,7 @@ The kubelet automatically tries to create a {{< glossary_tooltip text="mirror Po
 on the Kubernetes API server for each static Pod.
 This means that the Pods running on a node are visible on the API server,
 but cannot be controlled from there.
+The Pods' name will suffixed with the node hostname with a leading hyphen
```

Comment:
```suggestion
The Pod names will suffixed with the node hostname with a leading hyphen
```

Howdy, @Xorima! I'd recommend changing the possessive such that it reads like this suggestion, and provides a bit more clarity.